src/styles.css

```
@@ -1,709 +1,416 @@
+  --orange-border: #E2E8F0;
+  --ink: #0F172A;
+  --ink2: #334155;
+  --ink3: #475569;
+  --muted: #475569;
+  --border: #E2E8F0;
+  --border2: #E2E8F0;
+  --green: #0EA5E9;
+  --red: #DC2626;
+  --blue: #0EA5E9;
+  --purple: #6366F1;
+  --shadow: 0 1px 2px rgba(15, 23, 42, .06), 0 6px 18px rgba(15, 23, 42, .04);
```

Stylelint (17.11.0)

[error] Expected "rgba" to be "rgb" (color-function-alias-notation)

(color-function-alias-notation)